### DISCUSSION TITLE
Ha az adózó minden analitikus adatot átad, akkor miért kell még bevallás?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#37] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők!

Ha az ügyfél minden analitikus adatot átad, akkor miért kell még bevallás?

Ez redundánsnak tűnik.

A választ előre is köszönjük!

**Comment by renced42** [comment]
Kedves @szalaierika 

Ez leginkább a jogi környezet miatt van, meg a fogalmak miatt :)
Egyrészt bevallási kötelezettség van előírva, ezt a kötelezettséget az adózó önbevallásal teszi meg.
Másrészt az analitikából nem a NAV számolja ki a "Piros" számokat, hanem az ügyfél adja meg.
Viszont az analitikában megadott adatok alapján elkészül a bevallás tervezet, amit az ügyfél jóváhagy(hat) bizonyos validációkat követően.
Tehát aki gépileg adja át az analitikát valójában bevallást készít, neki más külön bevallást nem kell készíteni.

**Comment by szalaierika** [comment]
Köszi szépen a választ @renced42!

Abszolút értem. De nem racionális. Inkább több munka, mint kevesebb, ugyanis:

Ahhoz, hogy az adózó ellenőrizni tudja a bevallási javaslatot, neki is el kell azt készítenie és össze kell vetnie azzal. Ha azonos, akkor örülünk, hanem, akkor keressük a különbségeket, ami aggregált adatoknál nem lesz egyszerű. ;)

**Comment by renced42** [comment]
> Ahhoz, hogy az adózó ellenőrizni tudja a bevallási javaslatot, neki is el kell azt készítenie és össze kell vetnie azzal
Kedves @szalaierika 
Nincs mivel összevetnie, mert mint írtam nem a NAV ajánlja ki a bevallási adatokat, hanem az ügyfél közli azt a NAV-al.
Az ügyfél kap tételes prediktív információkat a NAV-től, amit ha akar fel/kihasznál, illetve az analitika feltöltésével, futni fognak különféle validációk a NAV oldalán amiről az adózó gépi interfészen szintén értesülhet.

**Reply by szalaierika** [reply]
Kedves @renced42 
Te is ezt ezt írod korábban, hogy az adózó bizonyos validációk után hagyhatja jóvá a bevallás tervezetet. Persze az adózó biztosra akar majd menni és szigorúan le fogja ellenőrizni a bevallás tervezetet. Véleményem szerint ez azt jelenti, hogy ő is elkészíti a saját verzióját.

Közben megnéztem még az eÁFA M2M Folyamt doksiban is, hogy mi található ezzel kapcsolatosan (15. old. teteje):
'Az adózó amikor jóváhagyja a bevallását, akkor nem az általa feltöltött adatállományt hagyja jóvá bevallásként, hanem az adóhivatal által visszamutatott adatállomány tartalmát. Ezért kliens oldalon fontos, hogy a QueryDeclarationDataResponse válaszüzenet tartalmát a felhasználó megismerje, és annak adattartalma alapján hozza meg a bevallás jóváhagyásáról a döntését.'

Ez is egybevág azzal, amit korábban írtam, sajnos...
>

**Comment by szalaierika** [comment]
Kedves @renced42 !

Nagyon köszönöm a legutóbbi válaszod. Még rágódnom kell rajta egy kicsit. De mindenképpen jelentkezem még ezzel kapcsolatosan.

**Comment by szalaierika** [comment]
Kedves @renced42 
a 2023.05.13-i válaszod nem találom már itt. Ez véletlen, vagy ki lett törölve?

Mindenesetre, még nem érzem, hogy megnyugtató választ kaptunk volna arra kérdésre, hogy az adózó hogyan tudja leellenőrizni, hogy a BevfeldData XML-ben kapott bevallási adatok helyességét.

Az alábbiakban van a gondolatmenetünk:

- A VATDeclareationData/VatAnalytics-ben megadjuk az áfa-sorokat bizonylatonként és a a declarationSummary-ben az összesítéseket (6 tag), ami nem a teljes bevallás.
- A QueryDeclarationData operációval visszakapott válaszban kapjuk meg a bevallási javaslatot a BevfeldData XML-ben ("BevfeldData: a feltöltött XML állomány alapján az adóhivatali algoritmus alapján képzett Bevfeld XML (ÁNYK nyomtatványba betölthető XML) base 64 bináris adata" 15. old teteje).
- Ezt az adózó csak akkor tudja felelősséggel jóváhagyni, ha meggyőződött az adóbevallás sorainak helyességéről.
- Azt pedig úgy tudja megtenni, ha ő maga is kiszámolja ezeket az összegeket és összeveti a kiajánlott bevallás soraival. Ha egyezik jóváhagyja, ha nem, akkor keresnie kell a különbözetet.

Szóval szerintünk ez az ellenőrzési lépés nem hagyható ki.

**Reply by NTCA-HM** [reply, marked as answer]
Kedves @szalaierika!

A kérdés alapvetően arra vonatkozott, hogy miért kell bevallást készíteni, ha az analitikus nyilvántartást az adózó a NAV rendelkezésére bocsátja.
Az erre adandó válasz, hogy nem kell, mi annyit várunk az adózóktól, hogy a benyújtott analitika alapján generált bevallást jóváhagyásukkal véglegezzék.
Ha az adózóban nincs meg a rendszerrel szembeni bizalom arra, hogy az a benyújtott analitika adataival egyezően állítja össze a bevallást, akkor értelemszerűen le fogja ellenőrizni az adatokat úgy, hogy saját maga is megképzi a bevallás egyes sorait. De ugyanez igaz lenne akkor is, ha a végleges analitika benyújtásával véget érne a folyamat.
Az eÁfa előnyei az online számla, a vámhatározatok és az online pénztárgépek adatainak adókódolt lekérdezési lehetőségében, valamint az ÁNYK nyomtatványkitöltőhöz képest kiterjedtebb validációs funkciókban rejlenek. Utóbbi nagyban hozzájárulhat a tartalmilag helyes bevallások benyújtásához.

**Reply by Szildiko** [reply]
2023-as post, érdeklődnék, hogy még 2026-ban is így van?
Akkor ha jól értem, ha felelősség teljes programot akarunk készíteni, akkor a bevfeld xml-t, le kell töltenünk mielőtt az ügyfél jóváhagyja, és meg kell jelenítenünk neki, hogy átnézhesse, hogy amit analitikán feltöltöttünk ugyanazt az adatot kapja vissza összesenben? és átnézés után, mondja azt hogy jóváhagyja a bevfeld xml-t

**Comment by szalaierika** [comment]
Kedves @renced42 !

Még egy kapcsolódó kérdésünk lenne ehhez a témához.

A SAFT-HU github projektben szintén hasonló módon az áfa-analitika kiváltotta volna az áfa-bevallást a megértésünk szerint.

Ez még cél a jövőben?

Előre is köszönjük!

**Comment by renced42** [comment]
> Kedves @renced42 !
> 
> Még egy kapcsolódó kérdésünk lenne ehhez a témához.
> 
> A SAFT-HU github projektben szintén hasonló módon az áfa-analitika kiváltotta volna az áfa-bevallást a megértésünk szerint.
> 
> Ez még cél a jövőben?
> 
> Előre is köszönjük!

Szerintem ez a kérdés egyenlőre függőben van.
